virtme_ng/run.py

```diff
@@ -835,9 +835,7 @@ def _make_remote(self, args, make_command):
                     ["fakeroot", "debian/rules", "clean"], quiet=not args.verbose
                 )
             check_call_cmd(
-                self._format_cmd(
-                    make_command + f" -j {self.cpus}" + " modules_prepare"
-                ),
+                make_command + [f"-j {self.cpus}", "modules_prepare"],
```

Comment:
maybe better to split the first arg:

    ["-j", self.cpus, "modules_prepare"],

BTW, not related to this commit, but `self.cpus` feels wrong, I guess it should be `-j$(nproc --all)"` like above, to get the CPU cores from the remote host, not the local ones.

(note that we could add `make_command.append("-j$(nproc --all)")` at the beginning of this function not to repeat it here and above.)

Reply:
Actually `modules_prepare` is always executed locally, even when the kernel is built remotely, so we could use `nproc --all`, but `self.cpus` is also correct.